CONTRIBUTING.md

```diff
@@ -32,4 +32,24 @@ Currently github workflow is used for automaticaly build test and release. Workf
 To merge your PR, please try to add a git tag in the format `VX.Y.Z` (e.g. V3.3.0) to trigger the release process in your fork and pass the compilation tests.
 
 ### WIKI
-Because of the limitations of the github wiki, it is difficult to create PR for changes of wiki. Please refer to the answer under: https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github and create a ticket for your modifications.
\ No newline at end of file
+Because of the limitations of the github wiki, it is difficult to create PR for changes of wiki. Please refer to the answer under: https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github and create a ticket for your modifications.
+
+### Debug
+New Arduino IDE(v2.3.1) support debug.
+Add debug_custom.json file in your sketch folder. Find more information in https://docs.arduino.cc/tutorials/mkr-wifi-1010/mkr-jlink-setup.
+Example using XMC1100:
+
+```
+{
```

Comment:
Yep, this is a cool feature. We should be able to generate this file automatically, under the assumption of Jlink in certain default installation path...
But okay, this is another (and future) topic that the CAN... :)

Reply:
I will ask julian and put a ticket for it